wpimath/src/main/java/edu/wpi/first/math/controller/PIDController.java

```diff
@@ -344,6 +344,20 @@ public boolean isContinuousInputEnabled() {
     return m_continuous;
   }
 
+  /**
+   * Sets the minimum and maximum contributions of the integral term based on a range.
+   * 
+   * <p>The internal integrator is clamped so that the integral term's contribution to the output
+   * stays between minimumIntegral and maximumIntegral. This prevents integral windup.
+   * 
+   * @param intergralRaduis The distance above and bellow 0.0 that the maximum and minimum contribution will be set to.
+   */
+  public void setIntegratorRange( double intergralRange ) {
```

Comment:
This has several typos. Also, the existing overload has clearer semantics than this new one.